### DISCUSSION TITLE
Override Admin Core React Components with Fallback Chain

### DISCUSSION THREAD

**Discussion by tsvetann** [medusajs/medusa#5093] [Feature Requests]
Greetings MedusaJS team,

I'd like to propose an enhancement to the MedusaJS Admin which provides developers with the ability to override core React components more seamlessly. This feature would function by resolving React component files by looking up multiple directories in a chain, with a fallback to a default package if the override isn't found. Something similar to [webpack-theme-resolver-plugin](https://www.npmjs.com/package/webpack-theme-resolver-plugin)

**Motivation:**

**Flexibility for Developers:** 
Allowing developers to override core components easily can lead to more tailored and customized solutions for different projects without altering the core codebase and ejecting from the original Admin

**Maintainability:** 
By having a systematic fallback chain, it reduces the risk of breakages. If an override fails, the system can fall back to a known-working component.

Thoughts?

Thanks!

**Comment by reiser** [comment]
Any solution here available? I cant find documentation how to overwrite react components of medusa.
